photo.go

```diff
@@ -65,6 +73,38 @@ func (p *ImgixPhoto) GetThumbnailForWidthAndHeight(w, h int) string {
 	return fullUrl.String()
 }
 
+func (p *ThumborRescaledPhoto) Slug() string {
+	//TODO: golang noob: find out how to make this common with ImgixRescaledPhoto
```

Comment:
It's an interface method that both need to implement but can't have any sort of classing system. halp?

Reply:
If you do the "parent struct" thing suggested above, you could move this up to the parent struct. But as with OOP, the parent struct won't have any knowledge of methods and attributes of the "child structs".

I'm trying to translate the OOP terms into Go as best as I can. But if I can't explain something clearly, let me know and we can have a live chat to clear it up.